<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frameNewGesture">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>521</width>
     <height>411</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="lblInfo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Új gesztus felvétele</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblUserGuide">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>521</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblGestureInputLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Gesztus neve:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtinputGestureName">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNameOK">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>320</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblCvImg">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>391</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>lblGestureInputLabel</zorder>
   <zorder>txtinputGestureName</zorder>
   <zorder>btnNameOK</zorder>
   <zorder>lblCvImg</zorder>
   <zorder>lblUserGuide</zorder>
   <zorder>lblInfo</zorder>
  </widget>
  <widget class="QFrame" name="frameBlue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>410</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>GestureBoard</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDescription">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>231</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>Leírás</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
